src/C4Client.cpp

```diff
@@ -85,10 +85,10 @@ void C4ClientCore::CompileFunc(StdCompiler *pComp)
 // *** C4Client
 
 C4Client::C4Client()
-	: fLocal(false), pNetClient(nullptr), last_lobby_ready_change(0), lastSound{} {}
+	: C4Client(C4ClientCore()) {}
```

Comment:
Why did you change that? Also, style guide mandates brace initialization.

Reply:
It actually makes sense, instead of initializing muted here too.